src/pkg/auditext/model/model.go

```diff
@@ -60,3 +62,18 @@ var EventTypes = []string{
 	"delete_robot",
 	"update_configure",
 }
+
+// OtherEventTypes defines the types of other audit log event types excludes previous EventTypes: create_artifact, delete_artifact, pull_artifact
+var OtherEventTypes = []string{
```

Comment:
We can use the full list of event types and exclude create_artifact, delete_artifact, and pull_artifact, rather than maintaining a separate array that needs to be updated whenever a new type is added.